src/modules/Channel/components/MessageInput/useHandleUploadFiles.ts

```diff
@@ -0,0 +1,84 @@
+import { useCallback } from 'react';
+import { Logger } from '../../../../lib/SendbirdState';
+import { SendMFMFunctionType } from '../../context/hooks/useSendMultipleFilesMessage';
+import useSendbirdStateContext from '../../../../hooks/useSendbirdStateContext';
+import { SendableMessageType, isImage } from '../../../../utils';
+
+/**
+ * The handleUploadFiles is a function sending a FileMessage and MultipleFilesMessage
+ * by the received FileList from the ChangeEvent of MessageInput component.
+ */
+
+const FILE_SIZE_LIMIT = 300000000; // 300MB
+
+interface UseHandleUploadFilesDynamicProps {
+  sendFileMessage: (file: File, quoteMessage?: SendableMessageType) => void;
+  sendMultipleFilesMessage: SendMFMFunctionType;
+  quoteMessage?: SendableMessageType;
+}
+interface UseHandleUploadFilesStaticProps {
+  logger: Logger;
+}
+export type HandleUploadFunctionType = (files: FileList) => void;
+
+export const useHandleUploadFiles = ({
+  sendFileMessage,
+  sendMultipleFilesMessage,
+  quoteMessage,
+}: UseHandleUploadFilesDynamicProps, {
+  logger,
+}: UseHandleUploadFilesStaticProps): Array<HandleUploadFunctionType> => {
+  const { config } = useSendbirdStateContext();
+  const uikitMultipleFilesMessageLimit = config?.uikitMultipleFilesMessageLimit ?? 15;
```

Comment:
no need for ?? 15?
[This PR](https://github.com/sendbird/sendbird-uikit-react/pull/728/files) should set default to 10.

Reply:
oh, this was for my self-testing, let me fix it!